src/core/scene/flags.js

```diff
@@ -10,6 +10,7 @@
 
         picking: true,              // Picking enabled
         clipping: true,             // User-defined clipping enabled
+        frontClippingOnly: false,        // Used to assist drawing clipping caps
```

Comment:
Do you find this feature useful for other cases? If not, since it needs modification anyway, I would like to remove this flag. Cuz currently in the plugin I'm using a custom shader instead of actually use this flag to affect `programSourceFactory`